frontend/src/components/news-page/News.tsx

```diff
@@ -1,3 +1,43 @@
-import React from "react";
+import React, { useState, useEffect } from "react";
 
-export const News: React.FunctionComponent = () => <></>;
+export const News: React.FunctionComponent = () => {
+  const [articlesList, setArticlesList] = useState<any[]>([]);
+
+  useEffect(() => {
+    const url = "https://www.reddit.com/r/whales/top.json?limit=50&t=month";
+
+    const fetchData = async () => {
+      try {
+        const response = await fetch(url);
+        const json: any = await response.json();
+        const articles = json.data.children;
+        setArticlesList(
+          articles.filter((article: any) => article.data.is_video)
+        );
+      } catch (error) {
+        console.log("error", error);
```

Comment:
Please do something more productive with this error! Just outputting it to the console isn't enough (also, for future reference, please use `console.error()` instead of `console.log()` for errors), we should also display it to the user.

On second thought, this is probably best included in the follow-up "styling and UX" ticket I told you guys to create - so please add "good error handling" to that!